src/vorta/utils.py

```diff
@@ -416,14 +417,27 @@ def format_archive_name(profile, archive_name_tpl):
     """
     hostname = socket.gethostname()
     hostname = hostname.split(".")[0]
+    fqdn = _getfqdn(hostname)
+    borg_version = os.getenv("BORG_VERSION")
+    if not borg_version:
+        borg_version = subprocess.run(['borg', '--version'], stdout=subprocess.PIPE).stdout.decode('utf-8')
+        borg_version = borg_version.split(' ')[1]
+    borg_version_tuple = tuple(borg_version.split("."))
```

Comment:
Vorta already queries borg version information on startup. Why not use that?

Reply:
> Vorta already queries borg version information on startup. Why not use that?

I get circular import error when importing `borg_compat` in utils.py:
`ImportError: cannot import name 'borg_compat' from partially initialized module 'vorta.utils' (most likely due to a circular import)`. This is probably why borg version is being queried again here.

Reply:
Yes, be aware of circular imports. Helper functions and modules should only be used by higher level. So maybe `format_archive_name()` should be moved elsewhere, since it uses helper functions now.